packages/ckb-sdk-utils/src/crypto/blake2b.ts

```diff
@@ -375,9 +378,24 @@ export const blake2b = (
 }
 
 export const ready = (cb: Function) => {
-  b2wasm.ready(() => {
-    cb()
-  })
+  const { window } = new Window()
+  const ua = window.navigator.userAgent.toLowerCase()
```

Comment:
Actually, `const ua = navigator?.userAgent.toLowerCase() ?? ''` could be enough.